lib/phoenix_html_helpers/form.ex

```diff
@@ -267,6 +267,22 @@ defmodule PhoenixHTMLHelpers.Form do
   def inputs_for(form, field) when is_atom(field) or is_binary(field),
     do: inputs_for(form, field, [])
 
+  @doc false
+  def inputs_for(%{impl: impl} = form, field, options)
+      when (is_atom(field) or is_binary(field)) and is_list(options) do
+    IO.warn(
+      "inputs_for/3 without an anonymous function is deprecated. " <>
+        "If you are using HEEx templates, use the new Phoenix.Component.inputs_for/1 component"
+    )
```

Comment:
I carried this deprecation over from `phoenix_html`.  Is it still deprecated?  If so, `inputs_for/2` will also need deprecated.

Reply:
Oh, now I think I get it. You want to bring a deprecated clause back? Then I am pretty sure it was removed on purpose, sorry for the confusion :(

Reply:
That's what I was trying to ask :laughing: The real issue is `inputs_for/2` still exists but doesn't work.

Reply:
The project where I discovered the error has already been updated to not use `inputs_for/2`, so I can update this PR to remove it altogether if that's your preference.

Reply:
Oh, I see it. Let's remove the two-arity one instead, good call!

Reply:
Updated the PR to remove `inputs_for/2`.